src/plugins/default/DuplicateCheckerPlugin.ts

```diff
@@ -28,8 +28,10 @@ export class DuplicateCheckerPlugin implements IEventPlugin {
       let hashCode = 0;
 
       while (error) {
-        if (error.stack_trace && error.stack_trace.length) {
-          hashCode = (hashCode * 397) ^ Utils.getHashCode(JSON.stringify(error.stack_trace));
+        hashCode += (hashCode * 397) ^ Utils.getHashCode(error.message);
```

Comment:
Do you think we should be checking the message length as well? Os should getHashCode method check that?